packages/grafana-data/src/transformations/matchers/valueMatchers/substringMatchers.ts

```diff
@@ -0,0 +1,45 @@
+import { Field, FieldType } from '../../../types/dataFrame';
+import { ValueMatcherInfo } from '../../../types/transformations';
+import { ValueMatcherID } from '../ids';
+
+import { BasicValueMatcherOptions } from './types';
+
+const isLikeValueMatcher: ValueMatcherInfo<BasicValueMatcherOptions> = {
+  id: ValueMatcherID.substring,
+  name: 'Is Substring',
+  description: 'Match where value for given field is a substring to options value.',
+  get: (options) => {
+    return (valueIndex: number, field: Field) => {
+      const value = field.values[valueIndex];
+      // eslint-disable-next-line eqeqeq
+      return value.includes(options.value);
```

Comment:
```suggestion
      return value.includes(options.value);
```

Reply:
^ it also seems like something is going wrong with this logic (value is null and is causing a runtime error in the transformation)